src/base.lua

```diff
@@ -7,6 +7,8 @@
 local appName, app = ...;
 app.EmptyTable = setmetatable({}, { __newindex = function() end });
 
+local GetAddOnMetadata = GetAddOnMetadata or C_AddOns.GetAddOnMetadata
```

Comment:
Let's do it like Blizzard and try the newer one first
https://github.com/Gethe/Aurora/blob/c70e555badcdb122fa2d9efd91542e5bef9227a2/libs/Shim.lua#L2
```suggestion
local GetAddOnMetadata = C_AddOns and C_AddOns.GetAddOnMetadata or GetAddOnMetadata
```